libraries/chef_server_org_provider.rb

```diff
@@ -3,17 +3,19 @@
 class Chef
   class Provider
     class ChefServerOrg < Chef::Provider::LWRPBase
-
+      provides :chef_server_org
       use_inline_resources if defined?(use_inline_resources)
 
       def whyrun_supported?
         true
       end
 
+      def chef_server_org; end
+
       action :create do
         execute 'create org' do
           command <<-EOM.gsub(/\s+/, ' ').strip!
-            chef-server-ctl org-create #{new_resource.org_name}
+            /bin/chef-server-ctl org-create #{new_resource.org_name}
```

Comment:
Why are we making the path explicit instead of relying on PATH being properly set?